### ISSUE TITLE
Support for 3D data for time series classification

### ISSUE THREAD

**Issue by shamilnabiyev** [hochschule-darmstadt/MetaAutoML#69]
Time series classification (TSC) libraries such as [McFly](https://github.com/NLeSC/mcfly), [sktime](https://github.com/alan-turing-institute/sktime) and [tslearn](https://github.com/tslearn-team/tslearn) expect 3D input data, also called Panel Data or Longitudinal Data. Each feature of 3D data contains a list of measurements over a defined time period (see the [issuecomment-1118360635](https://github.com/hochschule-darmstadt/MetaAutoML/issues/69#issuecomment-1118360635) below).

TODO:

* Evaluate the following file formats:
  - [x] [Attribute-Relation File Format: ARFF](https://www.cs.waikato.ac.nz/ml/weka/arff.html)
  - [x] [N-dimensional NumPy Array: npy](https://numpy.org/doc/stable/reference/arrays.ndarray.html)
  - [x] [sktime file format: ts](https://www.sktime.org/en/stable/examples/loading_data.html)
* Update the `ML_Ontology`
  - [x] add new `ML_Task` item:  `time_series_classification`
  - [x] add new `AutoML_solution` item: `mcfly`
  - [x] add new `dataset_type` to `Enum`: `time_series`
  - [x] add new mclfy task configurations to the `Configuration_item` sheet
  - [x] add new mclfy parameters  to the `Configuration_item` sheet
  - [x] add new config parameter sheet: `mcfly conig parameter (info)`
  - [x] add new cheatsheet : `AutoML cheatsheet mcfly (info)``
* Create a new data manager 
  - [x] Use the source code of `controller/managers/CsvManager.py` to create a new DataManager for the chosen 3D file format

**Comment by shamilnabiyev** [comment]
Try to use the source code of CsvManager: `controller/managers/CsvManager.py`

**Comment by shamilnabiyev** [comment]
3D Data in tabular form:

<img src="https://user-images.githubusercontent.com/22834492/166833474-a8dc2654-9205-46fa-af29-db007d0e6eb2.jpg" title="3D Tabular Data" alt="3D Tabular Data"/>

3D Data Plots:

<img src="https://user-images.githubusercontent.com/22834492/166832008-6f1dd5fd-f0b0-4939-bae7-b15ce78d1858.png" title="Longitudinal Data Plots" alt="Longitudinal Data Plots" width="500"/>

**Comment by shamilnabiyev** [comment]
Added new ml libraries:
```
:ML_library
	:sktime_lib
	:tslearn_lib
```

**Comment by shamilnabiyev** [comment]
Added new metrics:

```
:Metric
	:binary_accuracy
	:categorical_accuracy
	:sparse_categorical_accuracy
```

**Comment by shamilnabiyev** [comment]
Added new ML approaches:

```
:ML_approach
	:bag_of_sfa_symbols
	:individual_boss
	:contractable_boss
	:word_extraction_for_time_series_classification
	:multivariate_symbolic_extension
	:temporal_dictionary_ensemble
	:individual_temporal_dictionary_ensemble
	:elastic_ensemble
	:proximity_forest
	:proximity_tree
	:proximity_stump
	:hierarchical_vote_collective_of_transformation_based_ensembles_v1
	:hierarchical_vote_collective_of_transformation_based_ensembles_v2
	:time_series_combination_of_heterogeneous_and_integrated_embedding_forest
	:time_series_forest
	:supervised_time_series_forest
	:canonical_interval_forest
	:diverse_representation_canonical_interval_forest
	:random_interval_spectral_ensemble
	:shapelet_transform_classifier
	:arsenal_ensemble
	:canonical_time_series_characteristics
	:martrix_profile
	:signature_classifier
	:fresh_prince
	:summary_statistic
	:random_interval
	:inception_time
	:residual_neural_network
	:deep_convolutional_lstm
	:multi_scale_convolutional_neural_networks
	:inception_fully_convolutional_network	
```

**Comment by shamilnabiyev** [comment]
Added new AutoML solution:

```
:AutoML_solution
      :mcfly
```

**Comment by shamilnabiyev** [comment]
Added new config items:

```
:Config_item
	:use_approch_mcfly_time_series_classification
	:metric_mcfly_time_series_classification
	:loss_mcfly_time_series_classification
```

**Comment by shamilnabiyev** [comment]
Added new google sheet `AutoML cheatsheet mcfly (info)`

**Comment by shamilnabiyev** [comment]
Added new google sheet `mcfly config parameter (info)`

**Comment by shamilnabiyev** [comment]
Added new enum item

```
:Enum
   :time_series
```

**Comment by shamilnabiyev** [comment]
Added new configuration items

```
:Configuration_item
    :time_series_classification_in_mcfly
    :use_approch_mcfly_time_series_classification
    :metric_mcfly_time_series_classification
    :loss_mcfly_time_series_classification
```

**Comment by shamilnabiyev** [comment]
Added new ml libraries

```
:ML_library
    :sktime_lib
    :tslearn_lib
```

**Comment by shamilnabiyev** [comment]
Added new metrics.

```
:Metric 
	:balanced_accuracy
	:binary_accuracy
	:categorical_accuracy
```

**Comment by shamilnabiyev** [comment]
Merged into main #137
